tests/c/bfbug.c

```diff
@@ -0,0 +1,165 @@
+// Run-time:
+//   env-var: YKD_SERIALISE_COMPILATION=0
```

Comment:
This is the default so doesn't need to be set.

[It means the test won't test much useful yet, admittedly, as we already know.]

Reply:
Indeed. And setting it to `1` means it won't terminate in a reasonable amount of time (I'm pretty sure I had it running for at least 20 minutes), and thus might not be something we want bors to run each time.